build.sbt

```diff
@@ -24,8 +24,9 @@ lazy val compileScalastyle = taskKey[Unit]("compileScalastyle")
 lazy val testScalastyle = taskKey[Unit]("testScalastyle")
 
 val sparkVersion = "2.4.3"
-val hadoopVersion = "3.2.1"
+val hadoopVersion = "3.1.0"
```

Comment:
If force to use Hadoop 3.2.1, I think guava dependency still have version conflicts.

Reply:
Right. However, as guava is packaged into hive-exec jar, it's impossible to exclude guava from hive-exec jar.